### DISCUSSION TITLE
Moose install

### DISCUSSION THREAD

**Discussion by Draper18** [idaholab/moose#17464] [Q&A Getting Started] [answered]
Hello everyone,

I am trying to install MOOSE on Linux Ubuntu 20.04.2 LTS. I followed the install guide but when I got to run tests at the end they all crashed. In the troubleshooting guide I cannot even get the always ok test to not crash. While working through the trouble shooting guide I was able to get the hello world example to work though. I have tried deactivating and removing MOOSE then going back through the install guide only to end up with the same results. Any help is greatly appreciated.

Thanks,

Alex Draper

**Comment by GiudGiud** [comment]
Hello

Could you please post the error messages you are running into?

Guillaume

**Comment by Draper18** [comment]
Here is the error message when running the tests
[Moose Error messages.txt](https://github.com/idaholab/moose/files/6232701/Moose.Error.messages.txt)

**Comment by GiudGiud** [comment, marked as answer]
It seems you are missing libquadmath. `sudo apt install libquadmath` or [something close](https://www.devmanuals.net/install/ubuntu/ubuntu-12-04-lts-precise-pangolin/install-libquadmath0.html) should help
